homework_5/homework5.py

```diff
@@ -0,0 +1,229 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[182]:
+
+
+import pandas as pd
+import seaborn as sns
+from matplotlib.pyplot import *
+import numpy as np
+
+
+# # 1.REAL data
+
+# ## 1.1 GFF and BED readers
+
+# In[80]:
+
+
+def read_gff(file_path):
+    columns = ['chromosome', 'source', 'type', 'start', 'end', 
+               'score', 'strand', 'phase', 'attributes']
+    gff = pd.read_csv(file_path, sep='\t', names=columns)
+    gff = gff[~gff['chromosome'].str.contains('#')]
```

Comment:
Это будет корректно работать, но есть более удобный способ
```suggestion
    columns = ['chromosome', 'source', 'type', 'start', 'end', 
               'score', 'strand', 'phase', 'attributes']
    gff = pd.read_csv(file_path, sep='\t', names=columns, comment="#")
```